### ISSUE TITLE
Feature request: Add incomplete file suffix

### ISSUE THREAD

**Issue by antivirtel** [rakshasa/rtorrent#241]
Please implement that feature! The Transmission (eg. some.series.mkv.part) and uTorrent (eg. some.series.mkv.!ut) has it, so it is not impossible.

Thank you!

**Comment by ruda-almeida** [comment]
I would love it too!

**Comment by posledov** [comment]
:+1:

**Comment by jonesman** [comment]
I would like to see this feature too!

**Comment by jhelgert** [comment]
👍

**Comment by Gosokan** [comment]
:thumbsup:  :thumbsup:

**Comment by asavah** [comment]
+1

**Comment by trendless** [comment]
+1

**Comment by hboetes** [comment]
A use case for this is my sync software which I can instruct to ignore files with the .temp extension. syncing unfinished files it a waste of time.

**Comment by kantsdog** [comment]
+1

**Comment by reaperfromhell** [comment]
+1

**Comment by 130db** [comment]
+1

**Comment by maxcore** [comment]
no news about this ????

**Comment by Tuxie** [comment]
Please!

**Comment by MLWALK3R** [comment]
👍 Any chance of this feature being added this year ;)

**Comment by sapien2** [comment]
+1

**Comment by gjj2828** [comment]
+1

**Comment by mortis42** [comment]
Came here via Google search for this feature. Add my vote.

**Comment by gl0biz** [comment]
yes ! this is exactly what im missing from rtorrent! but seems like no developer wants to touch it. 

is this really so hard to make happen?

**Comment by feanorknd** [comment]
I do not understand why nobody add this simple feature, like every other bt client as being requested...
+1
;)

**Comment by ronioncloud** [comment]
It would be super, to move with rclone.

**Comment by docteurzoidberg** [comment]
+1
i know it can be done via rtorrent config but an option would be a lot nice. for syncing things

**Comment by Enissay** [comment]
Any reason not to implement this ? It's been pending since 2014!

**Comment by Drag0nFly** [comment]
Baffled to see this is still marked as open and unimplemented; had a number of files which were only partially downloaded but looked to be complete, according to their filenames, as they were scraped and playable. _(Impossible to distinguish between actual 100% downloaded files and in-progress downloads unless one does forensics on the filesystem-level.)_

Is there an unoffical patch somewhere to add this? I usally compile rtorrent (and rtorrent-PS from scratch anyway)
